.github/actions/monitor-cdk-verified-batches/batch_verification_monitor.sh

```diff
@@ -9,10 +9,17 @@ verified_batches_target="$1"
 # The script timeout (in seconds).
 timeout="$2"
 
+# Name of RPC service to query
+rpc_service="$3"
```

Comment:
nit: longer term lets consider adding explicit arg flags for readability and maintainability .. also, since we're adding additional flag, lets update the usage cmd above to reflect !

Reply:
I plan to revisit this one, I think adding an `rpc_url` flag would be better but for this PR it's fine.

Reply:
I will add a cmd usage in the script.